packages/profiles/source/wallet.ts

```diff
@@ -667,7 +667,8 @@ export class Wallet implements IReadWriteWallet {
 
 	#decimals(): number {
 		try {
-			return this.manifest().get(Coins.ConfigKey.CurrencyDecimals);
```

Comment:
For some reason data for `Coins.ConfigKey.CurrencyDecimals` was empty, so instead of that I directly extracted it from the config.